src/esi/VarState.cc

```diff
@@ -282,7 +282,7 @@ void
 ESIVarState::addVariable(char const *name, size_t len, Variable *aVariable)
 {
     String temp;
-    temp.limitInit (name, len);
+    temp.assign (name, len);
```

Comment:
Please fix formatting while changing this line.

```suggestion
    temp.assign(name, len);
```